distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+Files that could not be processed are moved to the `<${branding}_HOME>/etc/failed` directory.
+
+${branding} also monitors the `<${branding}_HOME>/etc` directory for any new `.config` file that gets added.
+As soon as a new file is detected, it is read, processed and moved to the appropriate directory based on whether it was successfully processed or not.
+
+==== Configuring Managed Service Factory Bundles
+
+Services that are created using a Managed Service Factory can be configured using `.config` files as well.
+The configuration files follow a different naming convention however.
+The files must start with the Managed Service Factory PID, be followed by a unique identifier and have a `.config` extension.
+For instance, assuming that the Managed Service Factory PID is `org.codice.ddf.factory.pid` and two instances of the service need to be configured, files `org.codice.ddf.factory.pid.uniqueID1.config` and `org.codice.ddf.factory.pid.uniqueID2.config` should be created and added to `<${branding}_HOME>/etc`.
+
+The unique identifiers used in the file names have no impact on the order in which the configuration files are processed.
+No specific processing order should be assumed.
+Also, a new service will be created and configured every time a configuration file matching the Managed Service Factory PID is added to the directory, regardless of the number used.
+
+These configuration files must also contain a `service.factoryPid` property set to the factory PID (without the sequential number).
+They should not however contain the `service.pid` property.
+
+===== File Format
+
+The basic syntax of the `.config` configuration files is similar to the older `.cfg` files but introduces support for lists and types other than simple strings.
+The type associated with a property must match the `type` attribute used in the corresponding `metatype.xml` file when applicable.
+
+The following table shows the format to use for each property type supported.
+
+[cols="1,2,4" options="header"]
+|===
+
+|Type
+|Format
+|Example
+
+|Service PID
+|`service.pid = "servicePid"`
+|`service.pid = "org.codice.ddf.security.policy.context.impl.PolicyManager"`
+
+|Factory PID
+|`service.factoryPid = "serviceFactoryPid"`
+|`service.factoryPid = "Csw_Federated_Source"`
+
+|Strings
+|`name = "value"`
+|`name = "john"`
+
+|Booleans
+|`name = B"true\|false"`
+|`authorized = B"true"`
+
+|Integers
+|`name = I"value"`
+|`timeout=I"60"`
+
+|Longs
+|`name = L"value"`
+|`diameter = L"10000"`
+
+|Floats
+|`name = F"value"`
+|`cost = F"10.50"`
+
+|Doubles
+|`name = D"value"`
+|`latitude = D"45.0234"`
+
+|Lists of Strings
+|`name = [ "value1", "value2", ... ]`
+|`complexStringArray = [ "{\"url\"\ \"http://test.sample.com\"\ \"layers\"\ [\"0\"]\ {\"VERSION\"\ \"1.1\|1.2\"\ \"image/png\"}\ \"beta\"\ 1}", "{\"url\"\ \"http://test.sample.com"\ 0.5}", "/solr\=SAML\|PKI\|basic", "/security-config\=SAML\|basic" ]`
```

Comment:
The example is really difficult to read. Could we try to change it a little and lay it out like this instead:

```
complexStringArray = [
    "{\"url\"\ \"http://site1.com\" [\"0\"]}",
    "{\"url\"\ \"http://site2.com\" [\"1\", \"2\"]}"
]
```